src/platforms/javascript/common/sourcemaps/troubleshooting_js.mdx

```diff
@@ -31,6 +31,18 @@ Double-check that your deployed, final JavaScript files have `sourceMappingURL`
 
 Alternately, instead of `sourceMappingURL`, you can set a `SourceMap` HTTP header on your minified file. If this header is present, Sentry will use it to discover the location of your source map.
 
+<PlatformSection notSupported={["javascript.electron"]}>
+
+## Verify artifacts distribution match value configured in your SDK
+
+Whenever you are using distribution identifier (`dist` configuration option in the SDK), it has to be matched during the source maps upload as well.
+To do this, use `--dist` flag in the `sentry-cli` or `dist` option if you are using `@sentry/webpack-plugin`, and make sure that the same value is used in both places.
+
+To verify that the distribution has been used and set correctly, open up the issue from the Sentry UI and check if the `dist` tag is present.
+For the artifacts, go to the `Source Maps` page in the project settings, and verify that required files have this value set. If not, it will say `none` instead.
```

Comment:
The style guide now puts all sentences in a paragraph on the same code line, for better wrapping. Also, I don't think we need to mention "none" if we've told them where to look - it's pretty self-explanatory.

```suggestion
To verify that the distribution has been set correctly in the SDK, open an issue in the Sentry UI and check that the `dist` tag is present. For artifacts, go to the `Source Maps` page in project settings, choose the release shown on the event you just checked, and verify that the `dist` value (in the small oval next to the upload time) matches that on the event.
```